lib/sedna/algorithms/aggregation/__init__.py

```diff
@@ -37,12 +37,18 @@ def aggregate(self, weights, size=0):
         :param size: numbers of sample in each loop
         """
 
+# os.environ['config_file'] = '/home/plato/examples/configs/server.yml'
+# from plato.servers import fedavg
 
 @ClassFactory.register(ClassType.FL_AGG)
 class FedAvg(BaseAggregation, abc.ABC):
     """Federated averaging algorithm"""
 
-    def aggregate(self, weights, size=0):
+    def __init__(self, model):
+        # self.fedavg_server = fedavg.Server(model=model)
+        pass
+
+    def aggregate0(self, weights, size=0):
```

Comment:
Why is naming `aggregate0` more recommended than `aggregate`?

Reply:
updated